src/utils.js

```diff
@@ -1,6 +1,10 @@
-const executeThunk = async (thunk, dispatch, getState) => {
+import { useMediaQuery } from 'react-responsive';
+
+export const executeThunk = async (thunk, dispatch, getState) => {
   await thunk(dispatch, getState);
   await new Promise(setImmediate);
 };
 
-export default executeThunk;
+export function useIsMobile() {
```

Comment:
useIsMobileOrLower?

Reply:
I checked from browser tablet screens are bigger than this that why I used only mobile term here

Reply:
got it.